modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -882,6 +884,7 @@ object SeqexecEngine extends SeqexecConfiguration {
         // TODO: Sequence completed event not emitted by engine.
         case SystemEvent.Completed(_, _, _, _)                                    => SequenceUpdated(svs)
         case SystemEvent.StopCompleted(id, _, _, _)                               => SequenceStopped(id, svs)
+        case SystemEvent.PartialResult(_, _, _, Partial(_: InternalPartialVal))   => NullEvent
```

Comment:
This is an extra to avoid sending messages for intermediate steps that don't concern the client